cmds/core/ls/ls_plan9.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build plan9
```

Comment:
you actually don't need the tag since the file ends in _plan9.go, I'll let you decide if you want to fix it :-)